core/src/main/java/de/danoeh/antennapod/core/storage/DBTasks.java

```diff
@@ -595,15 +599,15 @@ private static FeedItem searchFeedItemByIdentifyingValue(Feed feed,
                             item.getIdentifyingValue());
                     if (oldItem == null) {
                         // item is new
-                        final int i = idx;
                         item.setFeed(savedFeed);
                         item.setAutoDownload(savedFeed.getPreferences().getAutoDownload());
-                        savedFeed.getItems().add(i, item);
```

Comment:
If I'm not mistaken, this is a list of items of which values where changed and that have to be propagated to the database.
Therefor, adding the items at a particular position doesn't really make much sense.